extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -15,4 +15,11 @@
  */
 
  amp-sticky-ad {
+ 	position: fixed;
```

Comment:
nit: indent seems off. The `amp-sticky-ad {` should be flush against the edge. And the properties (`position:fixed`) should be two spaces from the edge. Please do not use tabs.